independent-projects/bootstrap/gradle-resolver/src/main/java/io/quarkus/bootstrap/resolver/model/QuarkusModel.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.bootstrap.resolver.model;
+
+import java.util.Set;
+
+public interface QuarkusModel {
+
+    Workspace getWorkspace();
+
+    Set<Workspace> getAdditionalWorkspace();
```

Comment:
As previously mentioned, there should only be one workspace that includes one or more projects or modules.